Dockerfile

```diff
@@ -28,4 +28,4 @@ COPY hack/git-ask-pass.sh /usr/local/bin/git-ask-pass.sh
 
 USER 1000
 
-ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/argocd-image-updater"]
+ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/argocd-image-updater"]
```

Comment:
Dockerfile has already been fixed recently in master. So no need to change it in this PR.